docs/contents/users/partners.md

```diff
@@ -0,0 +1,245 @@
+# Overview of Partners
+
+A number of partners are providing install methods as well a integrations
```

Comment:
I assume you want to move this page? `git mv` ?

Reply:
Good question. I left it in both places for now, since someone seemed to want it under the community heading and I wanted to put in in the install section. I'm fine removing it from its original location.

Reply:
I went ahead and removed the partners.md from the community directory and nav, so it only exists under install now.